README.md

```diff
@@ -110,6 +110,41 @@ possible by the use of the `UserService` service provided by
 You can see the details in
 [`packages/shopping/src/controllers/user.controller.ts`](https://github.com/strongloop/loopback4-example-shopping/blob/master/packages/shopping/src/controllers/user.controller.ts).
 
+### Authorization
+
+For an authorization demo in this example, all requests to the endpoint
+`/users/{userId}/orders` (i.e, `UserOrderController`) are validated for specific
+user access.
+
+Endpoint authorization is done using `@loopback/authorization` package and the
+provisions in it are implemented using
+[Casbin](https://github.com/casbin/casbin) configurations.
+
+The following is a typical scenario for an authorization demo:
+
+1. An existing user credentials is used to login using the endpoint
+   `/users/login`. The JWT token returned has the user name (lets say the user's
```

Comment:
`lets say` -> `let's say`

Reply:
I'm thinking maybe we can reword as follows:
```
Bob logs into the shopping app using his existing user credential via endpoint `/users/login`.  The JWT token.....
```

If we're going to this route, the following bullet would be in the form of "Bob wants to get his orders...." etc.

Reply:
that is correct, I will reword with a user based example in mind